projects/element-ng/dashboard/widgets/si-timeline-widget/si-timeline-widget.component.html

```diff
@@ -21,7 +21,7 @@
       @if (!linkValue.action) {
         <a [siLink]="linkValue">
           {{ linkValue.title | translate }}
-          <i class="link-icon link-end element-right-2 flip-rtl"></i>
```

Comment:
here as well, also element-right-2 should be loaded from the icons file as in other components